WordPress/Classes/Utility/Universal Links/Routes+Stats.swift

```diff
@@ -95,19 +95,19 @@ extension StatsRoute: NavigationAction {
                 showStatsForDefaultBlog(from: values, with: coordinator)
             }
         case .daySite:
-            showStatsForBlog(from: values, tab: .days, using: coordinator)
+            showStatsForBlog(from: values, tab: .traffic, unit: .day, using: coordinator)
         case .weekSite:
-            showStatsForBlog(from: values, tab: .weeks, using: coordinator)
+            showStatsForBlog(from: values, tab: .traffic, unit: . week, using: coordinator)
```

Comment:
```suggestion
            showStatsForBlog(from: values, tab: .traffic, unit: .week, using: coordinator)
```

Reply:
Thanks, strange that linter didn't catch it